Tests/BasicPerformanceTests/SortedArrayPerfTests.swift

```diff
@@ -21,4 +25,30 @@ class SortedArrayPerfTests: XCTestCasePerf {
             }
         }
     }
+
+    func testPerformanceOfSortedArrayInsertWithDuplicates() {
+        @inline(never)
+        func insert<T, S: Sequence>(_ sequence: S, into array: inout SortedArray<T>) where S.Iterator.Element == T {
```

Comment:
I had it like that initially, but I wanted it to be comparable to the old `insert(contentsOf:)` implementation. As I'm posting performance results here I'm happy to do that.